src/Api/Glances/Glance.php

```diff
@@ -32,17 +32,17 @@
 class Glance
 {
     /**
-     * @var Application Pushover application.
+     * @var Application pushover application
      */
     private $application;
 
     /**
-     * @var Recipient Pushover user.
+     * @var Recipient pushover user
      */
     private $recipient;
 
     /**
-     * @var GlanceDataFields Glance Data Fields.
+     * @var GlanceDataFields glance Data Fields
      */
```

Comment:
Is it possible to go through such changes, review them and move description to the top please? E.g.:

```
   /**
   * Glance Data Fields
   * 
   * @var GlanceDataFields
   */
```

Reply:
Will check in the next days

Reply:
This will be moved anyway when we add native types. From this point on it is superfluous for Cs-Fixer and it will be removed, resulting in the plain description as comment.